x-pack/plugins/stack_connectors/public/connector_types/stack/opsgenie/create_alert/translations.ts

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+export * from '../translations';
+
+export const MESSAGE_NOT_DEFINED = i18n.translate(
+  'xpack.stackConnectors.components.opsgenie.messageNotDefined',
+  { defaultMessage: '[message]: expected value of type [string] but got [undefined]' }
+);
+
+export const MESSAGE_NON_WHITESPACE = i18n.translate(
+  'xpack.stackConnectors.components.opsgenie.messageNotWhitespace',
+  { defaultMessage: '[message]: must be populated with a value other than just whitespace' }
+);
+
+export const LOADING_JSON_EDITOR = i18n.translate(
+  'xpack.stackConnectors.sections.ospgenie.loadingJsonEditor',
+  { defaultMessage: 'Loading JSON editor' }
+);
+
+export const MESSAGE_FIELD_LABEL = i18n.translate(
+  'xpack.stackConnectors.components.opsgenie.messageLabel',
+  {
+    defaultMessage: 'Message (required)',
+  }
+);
+
+export const DESCRIPTION_FIELD_LABEL = i18n.translate(
+  'xpack.stackConnectors.components.opsgenie.descriptionLabel',
+  {
+    defaultMessage: 'Description',
+  }
+);
+
+export const MESSAGE_FIELD_IS_REQUIRED = i18n.translate(
+  'xpack.stackConnectors.components.opsgenie.messageFieldRequired',
+  {
+    defaultMessage: '"message" field must be populated with a value other than just whitespace',
+  }
+);
+
+export const USE_JSON_EDITOR_LABEL = i18n.translate(
+  'xpack.stackConnectors.components.opsgenie.useJsonEditorLabel',
+  {
+    defaultMessage: 'Use JSON editor',
+  }
+);
+
+export const ALERT_FIELDS_LABEL = i18n.translate(
+  'xpack.stackConnectors.components.opsgenie.alertFieldsLabel',
+  {
+    defaultMessage: 'Alert fields',
+  }
+);
+
+export const JSON_EDITOR_ARIA = i18n.translate(
+  'xpack.stackConnectors.components.opsgenie.jsonEditorAriaLabel',
+  {
+    defaultMessage: 'JSON editor',
+  }
+);
+
+export const ENTITY_FIELD_LABEL = i18n.translate(
+  'xpack.stackConnectors.components.opsgenie.entityLabel',
+  {
+    defaultMessage: 'Entity',
+  }
+);
+
+export const TAGS_HELP = i18n.translate('xpack.stackConnectors.components.opsgenie.tagsHelp', {
+  defaultMessage:
+    'Type one or more custom identifying tags for this case. Press enter after each tag to begin a new one.',
+});
+
+export const TAGS_FIELD_LABEL = i18n.translate(
+  'xpack.stackConnectors.components.opsgenie.tagsLabel',
+  { defaultMessage: 'Opsgenie Tags' }
```

Comment:
Per https://elastic.github.io/eui/#/guidelines/writing/guidelines#capitalization, we should be using sentence case here:
```suggestion
  { defaultMessage: 'Opsgenie tags' }
```